test/lint/dummy_test.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2016 Google Inc.
```

Comment:
1. Avoid using ableist language, use 'placeholder' instead of 'dummy'. See https://developers.google.com/style/inclusive-documentation#ableist-language and https://source.android.com/setup/contribute/respectful-code#term-examples.
2. 2021 Google LLC